lib/defopt.py

```diff
@@ -91,6 +91,7 @@ def _ti_get_origin(tp):
         return {  # Py<3.7.
             typing.List: list,
             typing.Iterable: collections.abc.Iterable,
+            typing.Collection: collections.abc.Collection,
```

Comment:
`getattr(typing, "Collection", object())`

Reply:
Because typing.Collection doesn't seem to exist on Py3.5?

Reply:
Ahhh yeah it looks like it was introduced in 3.6. Sorry about that!